projects/components/src/header/page/page-header.component.ts

```diff
@@ -26,10 +27,17 @@ import { IconSize } from '../../icon/icon-size';
           <htc-label [label]="titlecrumb.label"></htc-label>
         </div>
       </div>
+
+      <ng-content></ng-content>
```

Comment:
What's this content expected to be?

Reply:
https://github.com/Traceableai/traceable-ui/pull/1155/files#diff-6ba90f6daf58f0b4c1af9e7debcfb5b9